### ISSUE TITLE
Не отображает одну метку из кластера на карте

### ISSUE THREAD

**Issue by DaryGor** [R1ZEN/react-yandex-maps#52]
Передаю в кластер массив меток. Из 6 штук на карте отображается 5. Хотя в дереве элементов DevTools есть все 6, но одна из них не распознается. Пробовала добавлять "фейковую метку", но при изменении маштаба карты построение сбивается и снова возвращает неверные метки. 

Карта расположена на сайте https://portland.forma.ru/infrastructure , можете отфильтровать метки парков и посмотреть, сколько элементов есть в DOM дереве, а сколько на карте. Всегда из массива удаляется именно первая метка.

**Comment by stereodenis** [comment]
@DaryGor может быть проблема с уникальным key?

**Comment by DaryGor** [comment]
> 
Нет, ключи задаются разные, в консоль тоже выводятся как уникальные 
<img width="732" alt="image" src="https://user-images.githubusercontent.com/49137878/218639210-bba3f21f-4bce-4c43-a6e3-65299e7008f2.png">

**Comment by stereodenis** [comment]
@DaryGor я бы в key добавил  `${el.coords}${index}`

**Comment by DaryGor** [comment]
> @DaryGor я бы в key добавил `${el.coords}${i}`

  Я попробовала, спасибо за рекомендацию, но пока не помогло (

**Comment by stereodenis** [comment]
@DaryGor пришлите что в `state.locations`

**Comment by poulezh** [comment]
@DaryGor у вас получилось решить проблему? столкнулся с похожей

**Comment by DaryGor** [comment]
@poulezh ДА! Но ошибка была глупейшая. Другой разработчик оказывается в CSS задал правило для неотображения первой метки, с установкой display none, и из дерева элементов это было неочевидно.

**Comment by poulezh** [comment]
> @poulezh ДА! Но ошибка была глупейшая. Другой разработчик оказывается в CSS задал правило для неотображения первой метки, с установкой display none, и из дерева элементов это было неочевидно.

спасибо за ответ, это хорошо, что просто решилось, у меня вообще проблема с кластеризацией меток. часть пропадает при зуме, если в фильтре выставлена категория любая. и нормально отображается только если все метки стоят на карте
